scripts/zones/Mhaura/npcs/Orlando.lua

```diff
@@ -8,12 +8,78 @@ package.loaded["scripts/zones/Mhaura/TextIDs"] = nil;
 -----------------------------------
 
 require("scripts/zones/Mhaura/TextIDs");
+require("scripts/globals/keyitems");
+require("scripts/globals/settings");
+require("scripts/globals/quests");
 
 -----------------------------------
 -- onTrade Action
 -----------------------------------
 
 function onTrade(player,npc,trade)
+    local QuestStatus = player:getQuestStatus(OTHER_AREAS, ORLANDO_S_ANTIQUES);
+    local itemID = trade:getItem();
+    local itemList =
+    {
+        564,   -- Fingernail Sack
```

Comment:
have the table store the gil too e.g.
```
local itemList = 
{
    {564, 200}, -- Fingernail Sack
}
```

then you can check the itemid, trade quantity, and gil rate all in one go instead of a 30 line if/elseif block

```
for _, item in pairs(itemList) do
    -- check item[1] == traded item's id
    -- check the quantity traded is 8
    -- set the ANTIQUE_PAYOUT var to GIL_RATE * item[2] (since item[2] is the gil value in our table for that item)
    -- trigger the event with taking the gil value from our table, and current itemid
end;
```